code/data_quality_tab/smr_audit.R

```diff
@@ -6,12 +6,9 @@ library(formattable)
 library(tidyverse)
 
 
-
-
 # National(Scotland) SMR Audit data --------------------------------------
 
-scot_accuracy <- read_csv(here::here("data", "Scotland_SMR_accuracy.csv"))%>%
-
+scot_accuracy <- read_csv("/conf/Data_Quality_Dashboard/data/Scotland_SMR_accuracy.csv")%>%
```

Comment:
The file paths here (and others in this script) will be affected by the decisions on the project structure and relative file paths.